config.py

```diff
@@ -1,6 +0,0 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-
-import cubeviz.data_factories
-from cubeviz import layout
-from cubeviz.listener import CubevizManager
-from cubeviz import startup
```

Comment:
Just out of curiosity, why were these removed?  Is it a change in glue?  I don't remember why I/we added them but I thought they needed to be in here.

Reply:
Oh, just saw the question as to why it was there. :).  NVM.  Not 100% sure, I don't remember

Reply:
config.py files are normally user configuration files - they shouldn't be in a package and aren't needed to run cubeviz.